core/modules/sink.h

```diff
@@ -5,6 +5,8 @@
 
 class Sink final : public Module {
  public:
+  Sink() { max_allowed_workers_ = Worker::kMaxWorkers; }
```

Comment:
Are there other modules this is true for?  I.e. other modules that are inherently thread safe?

Also, what is the right annotation for the various Queue* modules?